lib/Person.php

```diff
@@ -35,6 +35,7 @@
  */
 class Person extends ApiResource
 {
+
```

Comment:
Can you remove this empty line?

Reply:
Hey. I only added it because all the other classes have it. Should we remove it from all classes to be consistent? I know it's really nitpicking, but still.

Reply:
huh that is a weird one. I guess it does not hurt to leave as is on this one after all